kibana.md

```diff
@@ -181,11 +181,33 @@ $ bin/kibana-plugin remove readonlyrest_kbn
 To upgrade to a new version of a ReadonlyREST plugins for Kibana, you should:
 
 * [Unpatch Kibana](./#unpatching-kibana)
-* [Uninstall](./details/kibana-7.8.x-and-older.md#uninstall) the old plugin
-* Delete all the content of "optimize" directory (in the main Kibana installation directory) `rm -rf optimize/`
+* [Uninstall](#uninstalling) the old plugin
 * [Install](./#installation) the new one
 * [Patch Kibana](./#patching-kibana)
-* Restart Kibana.
+* Restart Kibana
+
+### Major version upgrades when using multi-tenancy
```

Comment:
who is creating tenant kibana indices? ROR KBN?
If so, it'd be nice to have automatic or semi-automatic migration. If not, maybe it's not strictly related to ROR and this section could be moved to a separate file?

Reply:
Yeah this is a good point. Kibana runs their migration tool at every boot (then eventually returns immediately if nothing is there to be done). We could have that invoked automatically as well, need to find a way to call it though.